mobile/native/src/ln_dlc/node.rs

```diff
@@ -291,14 +292,28 @@ impl Node {
             .send_message(node_id, msg.clone());
 
         // After sending the `CloseFinalize` message, we need to do some post-processing based on
-        // the fact that the DLC channel has been closed
+        // the fact that the DLC channel has been closed.
         if let Message::SubChannel(SubChannelMessage::CloseFinalize(SubChannelCloseFinalize {
             ..
         })) = msg
         {
-            let filled_order = order::handler::order_filled()?;
-            position::handler::update_position_after_dlc_closure(Some(filled_order))
-                .context("Failed to update position after DLC closure")?;
+            tracing::debug!("Checking if finalize subchannel closure is for position resize");
+
+            let positions = position::handler::get_positions()?;
```

Comment:
🔧 this should pnl my ever return a single position, if not we should probably fail?

However, I think it's safe to simply use first here.